docs/DeveloperGuide.md

```diff
@@ -45,42 +54,128 @@ title: PennyWise Developer Guide
+of several features of the application, allowing you to quickly familiarise yourself with the inner-workings of most parts of the code
+architecture and design.
+If you are **just getting started**, fret not! The guide gives a quick onboarding on how you can [set up and get started](#setting-up-getting-started).
+Once you have the development environment up and running, head over to the [design](#design) section to get a comprehensive
+overview of the code design of PennyWise.
+
+If you are in the **customer success team**, you may find the section on [requirements](#appendix-requirements) most helpful to
+you on _how_ and _why_ PennyWise came about. It allows you to have a thorough understanding of the [product](#product-scope),
+including the [user stories](#user-stories), [use cases](#use-cases) and [non-functional requirements](#non-functional-requirements).
+
+Whether you are a developer or in the customer success team, you may also find it useful to be aware of the [product requirements](#appendix-requirements)
+to ensure that we are always developing the finest application and solving the right problems for students. The [glossary](#glossary)
+is also helpful if you are unfamiliar with some terms used. Finally, the section on [testing](#appendix-instructions-for-manual-testing)
+provides information related to how you can test the application.
+
+<p align="right">
+    <a href="#top">Back to Top </a>
+</p>
+
+## How to use this Developer Guide
+These are some icons you may see throughout our developer guide.
+
+### Information Box
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+This provides some additional information that you are recommended to know.
+</div>
+
+### Tip Box
+
+<div markdown="block" class="alert alert-primary">:bulb: **Tip:**
+This provides some quick and convenient hacks that you might find useful.
+</div>
+
+### Danger Box
+
+<div markdown="block" class="alert alert-danger">:exclamation: **Warning**
+Danger zone! Do pay attention to the information here carefully.
+</div>
+
+### Formatting
+
+- `Highlights` are used to denote commands or output from the application.
+
+<p align="right">
+    <a href="#top">Back to Top </a>
+</p>
+
+If you are ready, let's [get started](#setting-up-getting-started)!
+
+--------------------------------------------------------------------------------------------------------------------
+
 ## **Acknowledgements**
 
-* {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
-  original source as well}
+* This project is based on the [AddressBook-Level3 project](https://github.com/se-edu/addressbook-level3) created by the [SE-EDU initiative](https://se-education.org).
+* Libraries used: [JavaFX](https://openjfx.io/), [Jackson](https://github.com/FasterXML/jackson), [JUnit5](https://github.com/junit-team/junit5)
+* Assets used:
+  * PennyWise Application Logo:
```

Comment:
Hi @JasonYapzx, can you apply a suggestion for this line? Thanks!

Reply:
Hey sflr, have added the logo website i used!